js_modules/dagster-ui/packages/ui-core/src/assets/RecentUpdatesTimeline.tsx

```diff
@@ -39,12 +39,25 @@ type Props = {
 };
 
 export const RecentUpdatesTimelineForAssetKey = memo((props: {assetKey: AssetKey}) => {
-  const data = useRecentAssetEvents(
```

Comment:
@salazarm hopefully a qq for you - on lines 32 and 33 above we define two types based on the return types of recentAssetEvents. Now that the events are returned as a single list, how to i get the right types here?

Reply:
`ReturnType<typeof useRecentAssetEvents>['data']`